server/views/partials/analytics/chartBaseHeatmap.njk

```diff
@@ -19,6 +12,44 @@ Required variables (in addition to base template):
   {%- endif -%}
 {%- endmacro %}
 
+{% macro valueColumn(column, columnIndex) %}
+  {{- column -}}
+{% endmacro %}
+
+{% macro rowTotalValue(total, rowIndex) %}
+  {{- total | thousands -}}
+{% endmacro %}
+
+{% macro rowValue(value, column, columnIndex) %}
+  {{- value | thousands -}}
+{% endmacro %}
```

Comment:
`columnIndex`, `rowIndex` and `column` are not used in these macros right? I'm guessing in other versions of these macros they're used? Or maybe some of these are not used anymore?

May be worth marking them as "unused" by prefixing them with an underscore, e.g. `_columnIndex` - assuming I'm not missing something here.

Reply:
hmmmm that wouldn't work. i call the macros with keyword arguments (as opposed to positional arguments) so cannot rename them! and i do that because 1) at the call site, you can see which argument is which and 2) order doesn't matter.

plus these ones are so short that you can see what they use and what they don't; this macro definition is the contract that compatible macros must obey.